.github/workflows/backend-cd.yml

```
@@ -20,63 +20,76 @@
-          --server.port=80 \
-          --DB_URL='${{ secrets.DEV_DB_URL }}' \
-          --DB_USERNAME='${{ secrets.DEV_DB_USERNAME }}' \
-          --DB_PASSWORD='${{ secrets.DEV_DB_PASSWORD }}' \
-          --ADMIN_TOKEN='${{ secrets.DEV_ADMIN_TOKEN }}' \
-          > /var/log/app.log 2>&1"
-          sleep 5
-          echo "Server started successfully"
+          echo "서버 시작 중... =$JAR_FILE"
+
+          sudo -E -b sh -c "java -jar '$JAR_FILE' > /var/log/app.log 2>&1"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)